native/package.json

```diff
@@ -122,9 +122,11 @@
     "jest-transform-stub": "^2.0.0",
     "jetifier": "^1.6.5",
     "metro-react-native-babel-preset": "^0.59.0",
+    "react-native-flipper": "^0.54",
```

Comment:
Should we use version 0.63 here? https://www.npmjs.com/package/react-native-flipper/v/0.63.0

https://github.com/jk-gan/redux-flipper/blob/master/package.json#L31

Reply:
The diff log says to use flipper v0.54: https://react-native-community.github.io/upgrade-helper/?from=0.63.2&to=0.63.3

I guess we should use the same version here

Reply:
@maxammann what do you think?

Reply:
oh alright :+1: